Google-Summer-of-Code-2023.md

```diff
@@ -0,0 +1,272 @@
+
+   - **Pro-tip!** Quality is more important than quantity; we want to see examples of your best work. So, please make sure to follow the [tips for success](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#tips-for-success), manually test your code before submitting (to ensure it does what you want it to and doesn't break anything else), ensure that your code conforms to the [style rules](https://github.com/oppia/oppia/wiki/Coding-style-guide), and pay attention to small details. These are good skills to learn when developing software in general, and they will also help you build credibility as a responsible developer who can be trusted to be a good steward of the Oppia codebase.
+
+5. Select one or more [GSoC project ideas](#oppias-project-ideas-list) that you're most interested in, and write your project proposal! We strongly encourage you to discuss your project ideas and share your proposal with the community, so that you can get feedback and ensure that what you're writing makes sense to others. Please follow the instructions in the [GSoC proposal template](#gsoc-proposal-template) section to do this.
+
+    If you have any questions about a project, or would like to discuss your approach with the Oppia community and get feedback, you can also email the [oppia-gsoc-discuss@](https://groups.google.com/g/oppia-gsoc-discuss) mailing list. Please be specific when asking questions, since this makes it easier for us to help you.
+    - **Pro-tip!** 75% of the accepted proposals had begun their work at least 4 weeks prior to the deadline. Hence, we strongly recommend that the contributors start writing proposals as early as possible.
+
+    Good luck!
+
+
+## FAQs
+
+**Q: Is Oppia Android a part of this year's GSoC?**
+
+A: Oppia Android won't be able to participate in GSoC this year. Note that this is a one-off, and Oppia Android hopes to be back in 2024 when we have better processes and documentation to help contributors get onboarded more smoothly.
+
+**Q: What technical skills do I need to work on Oppia?**
+
+A: For Oppia Web, Angular 2+, Python 3.8 and Google App Engine are useful and recommended for most Oppia work; in addition, UI design skills are useful for frontend, user-facing work. Please see the individual project ideas to determine whether these skills are recommended for the project in question.
+
+**Q: How can I increase my chances of getting selected?**
+
+A: Writing a good project proposal, engaging with the community, helping other contributors, successfully contributing, and demonstrating that you can work independently can all help you. We've also compiled some notes below on the [selection criteria](#selection-criteria) we'll be using this year.
+
+**Q: Can you be flexible around my other commitments in the summer?**
+
+A: No, Oppia will strictly adhere to GSoC timeline and will not be able to provide any additional extensions. Please refer to 
```

Comment:
```suggestion
A: No, Oppia will strictly adhere to GSoC timeline and will usually not be able to provide any additional extensions. Please refer to 
```

Just to give us a little wiggle room in case something extreme happens. Also, there shouldn't be a line break after "refer to"

Reply:
done. The line break was added to break the sentence. Visually when the wiki is added I don't think it will come over there 
<img width="514" alt="Screen Shot 2023-01-13 at 5 16 31 PM" src="https://user-images.githubusercontent.com/38973423/212429438-4df06834-a370-4f48-9451-b0a318c75937.png">

Reply:
Yes, the line break doesn't affect the rendered text. However, our style in the wiki is to not break lines in the middle of a paragraph. This avoids the creation of large diffs from re-wrapping blocks of text when just a few words are changed.

Reply:
Oh okay, will do that

Reply:
done